internal/manager/commands.go

```diff
@@ -72,6 +75,50 @@ var (
 			}
 		},
 	}
+
+	addTokenlistCmd = &cobra.Command{
```

Comment:
why duplicate `addTokenlistCmd` and `addTokenlistExtendedCmd` run command? it just a different file, create a function for it